clusters/build-clusters/02_cluster/storage-class/ssd_storageclass.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ssd
+parameters:
+  type: pd-ssd
+provisioner: kubernetes.io/gce-pd
```

Comment:
What for do we need this? What will use it?

Reply:
Deleted.

I was thinking to use ssd for PVCs for prometheus and alertManager.
But forgot to change it in the config.

AWS gp2 (0.10/G/Month) vs GCP ssd (0.17/G/Month).
Maybe we should start with GCP standard (0.40/G/Month)?

Reply:
In general storage is not very expensive thing in our infrastructure, unless it is `ebs/io1`.